# percentage of particles to use for the AMCL pose (int)
# "mean" state of the best x% particles. If x <= 0, the state with the highest weight is returned.
percentage_of_particles_to_use: 100

# Position
x_pos: 0.0
y_pos: 0.0
z_pos: 0.18

# Orientation
roll: 0.0
pitch: 0.0
yaw: 0.0

# Number of particles
particles: 300

# Standard deviations for movement model
/movement/x_std_dev: 0.15
/movement/y_std_dev: 0.15
/movement/z_std_dev: 0.05

/movement/roll_std_dev: 0.0
/movement/pitch_std_dev: 0.0
/movement/yaw_std_dev: 0.05


# Frames
mapFrame: "map"
worldFrame: "world"
baseFootprintFrame: "base_footprint"
baseLinkFrame: "base_link"

transform_tolerance_time: 1.0

# Observation parameters / Raycasting
# Sum of them must be 1
laser_z_hit: 0.60 # Mixture weight for the z_hit part of the model
laser_z_short: 0.10 # Mixture weight for the z_short part of the model
laser_z_rand: 0.10 # Mixture weight for the z_rand part of the model
laser_z_max: 0.20 # Mixture weight for the z_max part of the model

laser_sigma_hit: 0.02 # Standard deviation for Gaussian model used in z_hit part of the model
laser_lambda_short: 0.08 # Exponential decay parameter for z_short part of model
min_range: 0.01 # Laser min range
max_range: 14.0 # Laser max range

observation_threshold_trans: 0.2 # Minimun transform for a new observation
observation_threshold_rot: 0.4 # Minimum rotation for a new observation
sensor_sample_distance: 0.2 # Lidar point cloud subsampling
